packages/contract/script/pay/DeployDaimoPayAxelarBridger.s.sol

```diff
@@ -0,0 +1,135 @@
+                )
+            )
+        );
+
+        console.log("Axelar bridger deployed at address:", address(bridger));
+
+        vm.stopBroadcast();
+    }
+
+    function _getBridgeRoutes()
+        private
+        view
+        returns (
+            uint256[] memory chainIds,
+            DaimoPayAxelarBridger.AxelarBridgeRoute[] memory bridgeRoutes
+        )
+    {
+        // The bridge always gets sent to the DaimoPayAxelarBridger on the
+        // destination chain.
+        address axelarReceiver = CREATE3.getDeployed(
+            msg.sender,
+            keccak256("DaimoPayAxelarBridger-options1")
+        );
+
+        bool testnet = _isTestnet(block.chainid);
+        if (testnet) {
+            // Bridging not supported on testnet.
+            return (
+                new uint256[](0),
+                new DaimoPayAxelarBridger.AxelarBridgeRoute[](0)
+            );
+        }
+
+        if (
+            block.chainid == ARBITRUM_MAINNET ||
+            block.chainid == BASE_MAINNET ||
+            block.chainid == LINEA_MAINNET ||
+            block.chainid == OP_MAINNET ||
+            block.chainid == POLYGON_MAINNET
+        ) {
+            chainIds = new uint256[](1);
+            bridgeRoutes = new DaimoPayAxelarBridger.AxelarBridgeRoute[](1);
+
+            chainIds[0] = BSC_MAINNET;
+
+            for (uint32 i = 0; i < chainIds.length; ++i) {
+                uint256 fee = block.chainid == POLYGON_MAINNET
+                    ? 3000000000000000000 // 3.0 POLYGON
+                    : 300000000000000; // 0.0003 ETH
+
+                bridgeRoutes[i] = DaimoPayAxelarBridger.AxelarBridgeRoute({
+                    destChainName: _getAxelarChainName(chainIds[i]),
+                    bridgeTokenIn: _getAxlUsdcAddress(block.chainid),
+                    bridgeTokenOut: _getAxlUsdcAddress(chainIds[i]),
+                    bridgeTokenOutSymbol: "axlUSDC",
+                    receiverContract: axelarReceiver,
+                    fee: fee
+                });
+            }
+        } else if (block.chainid == BSC_MAINNET) {
+            chainIds = new uint256[](6);
+            bridgeRoutes = new DaimoPayAxelarBridger.AxelarBridgeRoute[](6);
+
+            chainIds[0] = ARBITRUM_MAINNET;
+            chainIds[1] = BASE_MAINNET;
+            chainIds[2] = ETH_MAINNET;
+            chainIds[3] = LINEA_MAINNET;
+            chainIds[4] = OP_MAINNET;
+            chainIds[5] = POLYGON_MAINNET;
+
+            for (uint32 i = 0; i < chainIds.length; ++i) {
+                bridgeRoutes[i] = DaimoPayAxelarBridger.AxelarBridgeRoute({
+                    destChainName: _getAxelarChainName(chainIds[i]),
+                    bridgeTokenIn: _getAxlUsdcAddress(block.chainid),
+                    bridgeTokenOut: _getAxlUsdcAddress(chainIds[i]),
+                    bridgeTokenOutSymbol: "axlUSDC",
+                    receiverContract: axelarReceiver,
+                    fee: 2_000_000_000_000_000 // 2 * 10^15 = 0.002 BNB
```

Comment:
maybe factor out into a function `getAxelarFeeByChainID(chainID)` that gets ~$1 in chain's native token?